src/main/kotlin/com/coder/toolbox/sdk/CoderRestClient.kt

```diff
@@ -133,7 +134,12 @@ open class CoderRestClient(
 
         retroRestClient =
             Retrofit.Builder().baseUrl(url.toString()).client(httpClient)
-                .addConverterFactory(MoshiConverterFactory.create(moshi))
+                .addConverterFactory(
+                    LoggingConverterFactory.wrap(
+                        context,
+                        MoshiConverterFactory.create(moshi).asLenient()
```

Comment:
I am not 100% sure what lenient exactly implies, but it seems dangerous to me because we rely on all of the data that is being parsed.  Like if an ID is missing for is malformed or example, would lenient let that pass?  Or say if the workspace status is missing.  At  best we show incorrect information, at worse we maybe do something we should not do (like allow a user to connect to a workspace that requires an update first).